symplyphysics/laws/kinematic/rotational_inertia_cylindrical_integral.py

```diff
@@ -0,0 +1,74 @@
+from sympy import Eq, Integral, pi
+from symplyphysics import (
+    units,
+    Symbol,
+    Quantity,
+    Function,
+    validate_input,
+    validate_output,
+    angle_type,
+    print_expression,
+)
+
+# Description
+## In case of a rigid body with a continuously distributed mass, its rotational inertia is expressed
+## as a volume integral over the entire body, i.e. a triple integral over space coordinates.
+
+# Law: I = Integral(rho(V) * r(V)**2, dV) = Integral(rho(r, theta, z) * r**3, dr*dtheta*dz)
+## I - rotational inertia
+## r - distance to axis of rotation
+## rho - density of volume element
+## dV = r * dr * dtheta * dz - volume element in cylindrical coordinates (r, theta, z)
+
+# Notes:
+## - The integration is carried out over the entire body as to include every volume element.
+## - THe z-axis is the rotational axis of the body
+
+rotational_inertia = Symbol("rotational_inertia", units.mass * units.length**2)
+density = Function("density", units.mass / units.length**3)
+
+radius = Symbol("radius", units.length)
+radius_start = Symbol("radius_start", units.length)
+radius_end = Symbol("radius_end", units.length)
+
+polar_angle = Symbol("polar_angle", angle_type)
+polar_angle_start = Symbol("polar_angle_start", angle_type)
+polar_angle_end = Symbol("polar_angle_end", angle_type)
+
+height = Symbol("height", units.length)
+height_start = Symbol("height_start", units.length)
+height_end = Symbol("height_end", units.length)
+
+law = Eq(
+    rotational_inertia,
+    Integral(
+        density(radius, polar_angle, height) * radius**3,
```

Comment:
Hm, cartesian and cylindrical versions should have the same form, but somehow they are different. It seems there is missing square of distance in cartesian form. It should be `distance(x, y, z)**2`
For volume elements we should use this helper: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/core/geometry/elements.py - it adds additional radius multiplier for cylindrical coordinates.

Reply:
Yes, but we should obtain it automatically. Especially useful to avoid mistake, like in cartesian version of this law.